File: tests/fixtures/language_adapter_symbols/Sample.cs
// HAND-DERIVED: standard C# syntax written from language knowledge, not from any adapter regex.
using System;

namespace Inventory
{
    public class Item
    {
        public string Name { get; set; }
        public int Quantity { get; set; }

        public void Print()
        {
            Console.WriteLine($"{Name}: {Quantity}");
        }
    }

    public interface IDescribable
    {
        string Describe();
    }
}

File: tests/fixtures/language_adapter_symbols/Sample.scala
// HAND-DERIVED: standard Scala syntax written from language knowledge, not from any adapter regex.
package com.example.inventory

case class Item(name: String, quantity: Int)

class Warehouse {
  private var items: List[Item] = List()

  def addItem(item: Item): Unit = {
    items = item :: items
  }
}

trait Describable {
  def describe(): String
}
